front/src/config/i18n/en.json

```diff
@@ -1609,6 +1610,16 @@
     "invalidIcon": "Icon is required",
     "sceneAlreadyExist": "A scene with that name already exist."
   },
+  "duplicateScene": {
+    "cardTitle": "Duplicate scene \"{{name}}\"",
+    "nameLabel": "Name",
+    "namePlaceholder": "Enter a new name for the duplicate scene, this must be different from the previous scene name.",
```

Comment:
This translation is very long (we can't see the full text). "duplicate" should be "duplicated"

Why not simply: 

> Enter a new name

?